scripts/lease.sh

```diff
@@ -20,3 +22,14 @@ delete_lease() {
   k8s-claimer --server=k8s-claimer-e2e.deis.com lease delete $TOKEN
   return $?
 }
+
+get-pod-logs() {
+  local components="deis-builder deis-database deis-minio deis-registry deis-router deis-controller"
```

Comment:
ok changed to be way more dynamic than just having some pre configured list. It will not capture all pods and get their current and previous logs and pipe them to the correct files.